### ISSUE TITLE
Show all month in a year?

### ISSUE THREAD

**Issue by fa10** [onefinestay/react-daterange-picker#163]
Hi,

I am trying to show all months of a year, but can get the `initialMonth` prop working it is starting from **February** instead of **January**.

```javascript
<DateRangePicker
     firstOfWeek={1}
     numberOfCalendars={12}
     selectionType='range'
     initialMonth={1}
     initialYear={2016}
     stateDefinitions={stateDefinitions}
     defaultState="available" />
```

**Comment by AlanFoster** [comment]
@fa10 if you supply the starting month of 0, does this work as expected?

**Comment by fa10** [comment]
No @AlanFoster it is not working when using **0**. I got it working by using `initialFromValue` and then supply `value` with a moment date.

**Comment by james-ff** [comment]
@AlanFoster Related to #195?

**Comment by AlanFoster** [comment]
This should be fixed in 2.0.1, please raise a new issue if there are any problems
